content/chapter2/2.2-chinese.md

```diff
@@ -33,7 +33,7 @@ void not_oops(int some_param)
 }
 ```
 
-相反的情形（期望传递一个非常量引用，但整个对象被复制）倒是不会出现，因为其不能编译。比如，你可能会尝试使用线程更新一个引用传递的数据结构：
+相反的情形（你想传递一个引用，但整个对象被复制过来）也会出现，比如你可能会在线程中更新一个以引用方式传递来过的数据结构：
```

Comment:
原文是这样的：
>It’s not possible to get the reverse scenario: the object is copied, and you wanted a
non-const reference, because this won't compile.

这里“相反的情况应该不会出现”

Reply:
It’s also possible to get the reverse scenario: the object is copied, and what you
wanted was a reference. This might happen if the thread is updating a data structure
that’s passed in by reference, for example:

Reply:
https://www.bogotobogo.com/cplusplus/files/CplusplusConcurrencyInAction_PracticalMultithreading.pdf

Reply:
我对比了本书的第一版和第二版。第一版的确是你写的那样没问题。不过，第二版这里做了修改，与第一版不同。

Reply:
https://livebook.manning.com/book/c-plus-plus-concurrency-in-action-second-edition/chapter-2/60  这个地址可以限时预览正版。
![捕获](https://user-images.githubusercontent.com/2756670/72438207-50241c80-37df-11ea-82d4-3e0a21123a3e.PNG)

Reply:
![](https://user-images.githubusercontent.com/2756670/72438241-603bfc00-37df-11ea-9222-fea07d81668f.PNG)